management-interface.go

```diff
@@ -555,19 +551,25 @@ func WifiNetworkHandler(w http.ResponseWriter, r *http.Request) {
 	err := error(nil)
 	if r.Method == http.MethodPost {
 		if err := r.ParseForm(); err != nil {
-			log.Print(err.Error())
+			log.Printf("WifiNetworkHandler error parsing form: %s", err)
```

Comment:
Maybe at least return a 400 error here rather than leaving the request unanswered